app/react/App/ErrorHandling/ErrorUtils.ts

```diff
@@ -0,0 +1,29 @@
+export interface RequestError extends Error {
+  title?: string;
+  summary?: string;
+  requestId?: string;
+  code?: string;
+}
+
+export interface APIError {
+  json: {
+    error?: string;
+    prettyMessage: string;
+    requestId?: number;
+  };
+  status?: number;
+}
+
+export const getRenderError = (apiResponse: APIError) => {
+  if (!apiResponse || !apiResponse.json) {
+    return {};
```

Comment:
@mfacar why is this case needed ? and if its actually needed at least the returned error should have a default name or message ? like the one below ? ```unexpected error```

Reply:
It was there just to add the condition, I moved to a function before calling